test/path_matcher_test.cc

```diff
@@ -77,6 +75,8 @@ std::ostream& operator<<(std::ostream& os, const Bindings& bindings) {
   return os;
 }
 
+}  // namespace
```

Comment:
Note: this is unrelated change, but it fixes pretty debug printing of Binding type in gtest (`operator<<` must be in the same namespace as the type it covers).